richtext-commonmark/src/commonMain/kotlin/com/halilibo/richtext/markdown/MarkdownRichText.kt

```diff
@@ -91,73 +91,79 @@ private fun RichTextScope.computeRichTextString(
     iteratorStack = iteratorStack.drop(1)
 
     if (!isVisited) {
-      val newFormatIndex = when (val currentNodeType = currentNode.type) {
-        is AstCode -> {
-          richTextStringBuilder.withFormat(RichTextString.Format.Code) {
-            append(currentNodeType.literal)
-          }
-          null
-        }
+      val parentLink = currentNode.links.firstParentOrNull<AstLink>()
+      val newFormatIndex =
+        when (val override = inlineContentOverride?.invoke(
+          this,
+          currentNode,
+          richTextStringBuilder,
+          if (parentLink == null) null else {{ onLinkClicked(parentLink.destination) }},
+        )) {
+          null -> when (val currentNodeType = currentNode.type) {
+            is AstCode -> {
+              richTextStringBuilder.withFormat(RichTextString.Format.Code) {
+                append(currentNodeType.literal)
+              }
+              null
+            }
 
-        is AstEmphasis -> richTextStringBuilder.pushFormat(RichTextString.Format.Italic)
-        is AstStrikethrough -> richTextStringBuilder.pushFormat(
-          RichTextString.Format.Strikethrough
-        )
+            is AstEmphasis -> richTextStringBuilder.pushFormat(RichTextString.Format.Italic)
+            is AstStrikethrough -> richTextStringBuilder.pushFormat(
+              RichTextString.Format.Strikethrough
+            )
 
-        is AstImage -> {
-          richTextStringBuilder.appendInlineContent(
-            content = InlineContent(
-              initialSize = {
-                IntSize(128.dp.roundToPx(), 128.dp.roundToPx())
-              }
-            ) {
-              if (contentOverride?.invoke(this@computeRichTextString, currentNode) {} != true) {
-                val parentDestination = currentNode.links.firstParentOrNull<AstLink>()?.destination
```

Comment:
I removed the one-off parent link handling for RemoteImage. It was actually fixed by the offset bug below.